### PR TITLE
Update installation guide in the documentation

### PR DESCRIPTION
**Pull request by yuyu2172**
I considered adding a docker file, but I didn't because it was tricky to setup GPUs and matplotlib.
Instead, I added more detailed steps to install ChainerCV using anaconda.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -7,16 +7,49 @@ ChainerCV is a **deep learning based computer vision library** built on top of `
...
+
+Pip
+~~~
 
-Please install `Chainer <https://github.com/pfnet/chainer/>`_ first.
 You can install ChainerCV using `pip`.
 
 .. code-block:: shell
 
     pip install chainercv
```
Comment (Hakuyume) [inline docs/source/index.rst]: Users cannot install ChainerCV by this instruction.
```
$ pip install chainer
$ pip install chainercv
Collecting chainercv
  Using cached chainercv-0.4.7.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-i4rdgwhe/chainercv/setup.py", line 6, in <module>
        from Cython.Distutils import build_ext
    ModuleNotFoundError: No module named 'Cython'
    
    ---------------------------------------
```

File: environment.yml [hunk 2]
```diff
@@ -9,6 +9,5 @@ dependencies:
   - opencv3=3.2.0
   - matplotlib
   - numpy
-  - scikit-learn
   - pip:
-    - chainer
+    - chainer=1.24
```
Comment (Hakuyume) [inline environment.yml]: How about adding `pillow`?

File: docs/source/index.rst [hunk 3]
```diff
@@ -7,16 +7,51 @@ ChainerCV is a **deep learning based computer vision library** built on top of `
...
+
+Pip
+~~~
 
-Please install `Chainer <https://github.com/pfnet/chainer/>`_ first.
 You can install ChainerCV using `pip`.
 
 .. code-block:: shell
 
+    # If Cython has not been installed yet, install it by commands like
```
Comment (Hakuyume) [inline docs/source/index.rst]: `pip install Cython` is a single command. How about `install it by the following command`?

Comment (yuyu2172) [inline docs/source/index.rst]: I wanted it to imply that `pip install Cython` is not the only way to install Cython.
People install them using apt as an alternative to pip for instance.

Comment (Hakuyume) [inline docs/source/index.rst]: I see.

**Comment by Hakuyume** [comment]
LGTM

**Pull request merged**